src/video/uikit/SDL_uikitmodes.m

```diff
@@ -270,7 +270,8 @@ static int UIKit_AddSingleDisplayMode(SDL_VideoDisplay *display, int w, int h,
     }
 
     mode.format = SDL_PIXELFORMAT_ABGR8888;
-    mode.refresh_rate = (int)UIKit_GetDisplayModeRefreshRate(uiscreen);
+    mode.refresh_rate_numerator = (int)UIKit_GetDisplayModeRefreshRate(uiscreen);
```

Comment:
UIKit_GetDisplayModeRefreshRate() returns a floating point value, this should be converted to numerator/denominator.

Reply:
actually the value is an int: 
`uiscreen.maximumFramesPerSecond`
https://developer.apple.com/documentation/uikit/uiscreen/2806814-maximumframespersecond ?

Reply:
It's an `NSInteger` which is 64-bit on most platforms (although `UIKit_GetDisplayModeRefreshRate` returns `NSUInteger` for some reason), so this is casting from 64 bit uint to 32 bit int which seems fine to me.